docs/assessment/index.md

```diff
@@ -105,16 +153,57 @@ An assessment is broken down in to a list of zones, like this:
 }
 ```
 
-- Each zone appears in the given order in the assessment. Zone titles are optional and are displayed to the student if present.
+| Zone Property   | Type    | Description                                                                                                                                           |
```

Comment:
A screenshot highlighting what a zone is would be helpful, or more text about what a zone is would be good.

Reply:
Want to add that in another PR?

Reply:
#12417